### DISCUSSION TITLE
GUI editor

### DISCUSSION THREAD

**Discussion by khaleeek** [espanso/espanso#1453] [Feature requests and Ideas]
![gui](https://user-images.githubusercontent.com/84661890/202832480-28ca3d03-6be7-43aa-9f10-82c66f4e5f48.png)

something this type of GUI for windows or linux to edit file becuse its not easy to edit file for lot peoples 
just hit GUI and add new expender shortcuts
its a nice idea.

**Comment by beedell-roke** [comment]
https://reddit.com/comments/gomu73

**Comment by tomkpt** [comment]
Its must need GUI Feature
